cve_lookup.py

```diff
@@ -23,8 +23,10 @@ def parse_dbs(folder):
     root = dict()
     header_printed = False
     for filename in glob.glob(folder + '/*.json'):
-        with open(filename) as ff:
-            cve_dict = json.loads(ff.read())
```

Comment:
The encoding change will (probably) fix this, but I'd also change it to use `json.load(ff)`, which will read the file object directly instead of reading it then passing it to `json.loads`

Reply:
See latest!